garrysmod/gamemodes/base/gamemode/npc.lua

```diff
@@ -3,6 +3,8 @@
 util.AddNetworkString( "PlayerKilledNPC" )
 util.AddNetworkString( "NPCKilledNPC" )
 
+local worldspawn = Entity(0)

```

Comment:
Just call game.GetWorld() at the time the hook is ran - this optimisation is not needed. Also the variable names should be changed to reflect the fact that they're an entity now, not a class string.